### ISSUE TITLE
Issue with finetuning model on custom dataset

### ISSUE THREAD

**Issue by ghost** [matterport/Mask_RCNN#2044]
Hi all,

I am trying to train a model with mask_rcnn_coco.h5 as the base model. However, the output produced by the trained model is complete garbage.  

**Case 1**: I annotated 5 or so images and trained a model on it for 67 epochs. And here is the last line of the log:

` 684/1000 [===================>..........] - ETA: 10:09 - loss: 0.0150 - rpn_class_loss: 5.0984e-05 - rpn_bbox_loss: 5.0718e-05 - mrcnn_class_loss: 9.9723e-04 - mrcnn_bbox_loss: 1.3671e-04 - mrcnn_mask_loss: 0.0137`

Loss is clearly very low, so I expect it to give good results AT LEAST on the training images (overfitting). But it produces on garbage or NONE on training images itself.

**Case 2:** 
I thought maybe I am using very small dataset (~5 images). Training on somewhat larger dataset should give good results. So, I started training on COCO VAL dataset which has 5k images. I trained for 40 epochs. I wasn't expecting very accurate model, but at least it should give 40-50% accuracy. But most of the time it generates random masks or detects no instances.

Here is the last line of the log:
`1000/1000 [==============================] - 1995s 2s/step - loss: 2.7258 - rpn_class_loss: 0.2099 - rpn_bbox_loss: 0.7793 - mrcnn_class_loss: 0.6342 - mrcnn_bbox_loss: 0.5556 - mrcnn_mask_loss: 0.5468 - val_loss: 2.7712 - val_rpn_class_loss: 0.1800 - val_rpn_bbox_loss: 0.7944 - val_mrcnn_class_loss: 0.7132 - val_mrcnn_bbox_loss: 0.5553 - val_mrcnn_mask_loss: 0.5282
`

If I look at the losses in tensorboard, all losses are decreasing over time with some of them reaching ~0.
Also, I noticed that even though I am using a pre-trained model, the starting loss is around 75. I am not sure if that's normal.

I am unable to figure out what's going wrong. Any leads/help would be appreciated.

Thanks

**Comment by 1chimaruGin** [comment]
@san999p  mrcnn_bbox_loss and mrcnn_mask_loss are so high

**Comment by ghost** [comment]
@1chimaruGin Thanks for the information. I figured out the issue, I mistakenly commented out the model loading line. So, it was not using the pre-trained model at all.
